t/t1008-mf-priority-update.t

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+test_description='Test multi-factor priority plugin order with no ties'
+
+. `dirname $0`/sharness.sh
+MULTI_FACTOR_PRIORITY=${FLUX_BUILD_DIR}/src/plugins/.libs/mf_priority.so
+SUBMIT_AS=${SHARNESS_TEST_SRCDIR}/scripts/submit_as.py
+PRIORITY_UPDATE=${SHARNESS_TEST_SRCDIR}/../src/cmd/flux-account-priority-update.py
+FLUX_ACCOUNT=${SHARNESS_TEST_SRCDIR}/../src/cmd/flux-account.py
```

Comment:
> An alternative would be to set `FLUX_EXEC_PATH` to include `${SHARNESS_TEST_SRCDIR}/../src/cmd`, then you can just run `flux account` _and_ `flux account-priority-update` as would be used after installation (`flux(1)` will take care of running `flux python CMD.py` for you...)

TIL! In my opinion that really cleans up the sharness tests and really helps make it clear what commands are being run in what tests; thank you for pointing this out. I ended up pushing a change that uses your suggestion of setting `FLUX_EXEC_PATH`. Now a test command actually looks like something we would see after installation:

```console
$ flux account-priority-update -p $(pwd)/FluxAccountingTest.db
```

I think this could help improve at least a couple of the other sharness tests I have in flux-accounting, but that's probably a better fit in another PR...